go/vt/vttablet/tabletserver/tabletserver_test.go

```diff
@@ -2232,6 +2235,7 @@ func setupTabletServerTest(t *testing.T, keyspaceName string) (*fakesqldb.DB, *T
 
 func setupTabletServerTestCustom(t *testing.T, config *tabletenv.TabletConfig, keyspaceName string) (*fakesqldb.DB, *TabletServer) {
 	db := setupFakeDB(t)
```

Comment:
same suggestion as given above:
Just like fakesqldb.New(t), should we have fakesqldb.NewWithVtSchema(nil) where we return db with added init queries and replace .New with .NewWithVtSchema where ever needed. 
setupFakeDB --> setupFakeDBForVtSchema